core/mondoo-gcp-security.mql.yaml

```diff
@@ -381,12 +411,22 @@ queries:
           ```bash
           gcloud compute instances add-metadata INSTANCE_NAME --metadata block-projectssh-keys=TRUE
           ```
+  - uid: gcp-compute-block-project-wide-ssh-keys-enabled-vm-instances-all
+    filters: asset.platform == "gcp" || asset.platform == "gcp-project"
+    mql: |
+      gcp.compute.instances.all(
```

Comment:
is it possible to write metadata.block-project-ssh-keys?

Reply:
no, this runs into an error (underscores `_` work, but hyphens `-` don't)